airbyte-protocol/models/src/main/java/io/airbyte/protocol/models/CatalogHelpers.java

```diff
@@ -63,7 +63,7 @@ public static ConfiguredAirbyteStream createConfiguredAirbyteStream(String strea
   }
 
   public static ConfiguredAirbyteStream createConfiguredAirbyteStream(String streamName, List<Field> fields) {
-    return new ConfiguredAirbyteStream().withStream(new AirbyteStream().withName(streamName).withJsonSchema(fieldsToJsonSchema(fields)));
+    return new ConfiguredAirbyteStream().withStream(new AirbyteStream().withName(streamName).withJsonSchema(fieldsToJsonSchema(fields))).withSyncMode(SyncMode.FULL_REFRESH);
```

Comment:
@cgardens is it fine to make this assumption for tests that were previously not setting this value at all?